public/js/constants/error-codes.js

```diff
@@ -0,0 +1,17 @@
+export const ALREADY_SUBSCRIBED = 'ALREADY_SUBSCRIBED';
+export const API_SIGN_IN_FAILURE = 'API_SIGN_IN_FAILURE';
+export const API_SIGN_OUT_FAILURE = 'API_SIGN_OUT_FAILURE';
+export const BRAINTREE_TOKENIZATION_ERROR = 'BRAINTREE_TOKENIZATION_ERROR';
+export const BRAINTREE_TOKEN_GET_FAILED = 'BRAINTREE_TOKEN_GET_FAILED';
+export const FXA_SIGN_IN_FAILURE = 'FXA_SIGN_IN_FAILURE';
+export const FXA_TOKEN_SIGN_IN_FAILURE = 'FXA_TOKEN_SIGN_IN_FAILURE';
+export const ONE_TIME_PAYMENT_FAILED = 'ONE_TIME_PAYMENT_FAILED';
+export const PAY_METHOD_DELETION_FAILED = 'PAY_METHOD_DELETION_FAILED';
+export const PAY_METHOD_GET_FAILED = 'PAY_METHOD_GET_FAILED';
+export const PRODUCT_ID_INVALID = 'PRODUCT_ID_INVALID';
+export const SUBS_BY_PAY_METHOD_GET_FAILED = 'SUBS_BY_PAY_METHOD_GET_FAILED';
+export const SUBS_GET_FAILED = 'SUBS_GET_FAILED';
+export const SUB_CREATION_FAILED = 'SUB_CREATION_FAILED';
+export const SUB_UPDATE_FAILED = 'SUB_UPDATE_FAILED';
+export const TRANSACTIONS_GET_FAILED = 'TRANSACTIONS_GET_FAILED';
+export const UNSPECIFIED_ERROR = 'UNSPECIFIED_ERROR';
```

Comment:
Yeah, I can kill that now it throws. I thought it made sense to not have an error with no code.